### PR TITLE
Update Tabs.tsx

### PR DESCRIPTION
**Pull request by RISHIKESHk07**
UPDATED THE Tabs.tsx with TSDoc comments
issue #976 
Signed-off-by: RISHIKESHk07 <rishikeshkuppala@googlemail.com>

### CODE REVIEW

**Comment by RISHIKESHk07** [comment]
@anshu8912 any changes ? need some help on the checks not sure how to resolve it .....

**Review by joaquimrocha — CHANGES_REQUESTED** [review]
Thanks for this PR @RISHIKESHk07 .
Please rename the commit as: `frontend common/Tabs.tsx: Add code docs`

**Comment by RISHIKESHk07** [comment]
Hey @joaquimrocha , sorry for the late reply i did change the commit message , anything else i should do the checks are failing , not sure where it went wrong

**Comment by joaquimrocha** [comment]
> Hey @joaquimrocha , sorry for the late reply i did change the commit message , anything else i should do the checks are failing , not sure where it went wrong

Hi @RISHIKESHk07 . Yes. The DCO is failing because the commit needs to be signed off by you. That's done by using `git commit -s`. It has to be signed off by an email that you have associated with your account on Github.

And now you also need to rebase this branch.

**Comment by RISHIKESHk07** [comment]
@joaquimrocha thanks for the help , did the changes , anything i need to correct?

**Comment by joaquimrocha** [comment]
You have one extra commit which I believe is due to not rebasing or a problem in doing so. Can you make sure that only your single commit is in the PR?

**Comment by RISHIKESHk07** [comment]
@joaquimrocha i had to merge the original repo with my main branch (changes were on the main  ), had some conflicts with @yolossn commit I did resolve it , any idea what I should try doing , thanks

**Comment by RISHIKESHk07** [comment]
Got this for the lint (make frontend-lint command ),
Oops! Something went wrong! :(

ESLint: 8.56.0

ESLint couldn't find the config "@kinvolk" to extend from. Please check that the name of the config is correct.

The config "@kinvolk" was referenced from the config file in "/home/intelliagent-19/headlamp/frontend/package.json".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

make: *** [Makefile:80: frontend-lint] Error 2

**Comment by RISHIKESHk07** [comment]
@joaquimrocha thanks , got it to work

**Comment by joaquimrocha** [comment]
Hmm, the test failed because you don't have a kubeconfig file?
What OS are you using?

**Comment by RISHIKESHk07** [comment]
@joaquimrocha  linux mint 20.3

**Comment by RISHIKESHk07** [comment]
@joaquimrocha any doc on the website i could follow ....

**Comment by yolossn** [comment]
> @joaquimrocha thanks , got it to work , i tried to make backend-test as well but got a error because i did not have the config file under /.kube , how can i set this up ?

This is happening because the backend is expecting a kubeconfig in the directory. To solve for this install minikube or other local kubernetes solutions like KIND. On a side note, Since you didn't make any changes to the backend, I don't think there is a need to run the backend-test locally.

File: frontend/src/components/verticalPodAutoscaler/VPAList.stories.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Meta } from '@storybook/react';
+import { Meta } from '@storybook/react/types-6-0';
```
Comment (yolossn) [inline frontend/src/components/verticalPodAutoscaler/VPAList.stories.tsx:1]: From the logs of the failing tests looks like this change is the issue. Either use the old package itself or include the changes to package.json also in the commit.

**Comment by joaquimrocha** [comment]
@RISHIKESHk07 , the commit is still looking like it's not the changes you wanted to add. Do you need help with this? If you submit a new branch, regardless of the format, we can also take it ourselves and redo the commit, setting you as co-author.

**Comment by RISHIKESHk07** [comment]
@joaquimrocha i found that my commit got merged with a @yolossn  commit so it was creating a problem , maybe its better to create a new PR ? , Would that be problem ?

**Comment by joaquimrocha** [comment]
> @joaquimrocha i found that my commit got merged with a @yolossn commit so it was creating a problem , maybe its better to create a new PR ? , Would that be problem ?

Hi @RISHIKESHk07 . The problem was not the PR, it was the commit. If you create a new commit and force push it to the branch you had created, it'd be fixed as well. But it is also fine to do it in a new PR.

**Pull request closed without merging**